renku/cli/dataset.py

```diff
@@ -90,16 +90,18 @@
 
 @click.group(invoke_without_command=True)
 @click.option('--datadir', default='data', type=click.Path(dir_okay=True))
-@pass_local_client(clean=True, commit=True)
+@pass_local_client(clean=False, commit=False)
```

Comment:
Does it maybe make more sense to put this as a default value to the pass_local_client definition or would that break to much stuff? Is there a reason why bool should have 3 possible states to handle in this case?

Reply:
Eventually, we should unify arguments and defaults on `pass_local_client` and `client.transaction`.